### DISCUSSION TITLE
Audio "Normalization"?

### DISCUSSION THREAD

**Discussion by Sclafus** [ppy/osu#25584] [Ideas] [answered]
Hello everyone,
Due to the nature of the game, some mp3's have a louder volume than others. This results in players having to change their music volume from song to song.
A normalizer (actually, a [Dynamic Range Compressor](https://en.wikipedia.org/wiki/Dynamic_range_compression)) could be a nice addition to maintain the volume of the music constant in between maps.
What do you think?

**Comment by Gabixel** [comment, marked as answer]
[Already planned(?)](https://github.com/ppy/osu/issues/1600) (got the link from [another post asking for something similar](https://github.com/ppy/osu/discussions/23517))
I assume they also want to normalize some volumes in general with this, [like hitsounds](https://github.com/ppy/osu/discussions/22384#discussioncomment-4762984).
